string skill_name
---
int8 response
